### PR TITLE
Remove the test of fluid StepDecay in set_lr_scheduler

### PR DESCRIPTION
**Pull request by zhengqiwen1997**
### PR types

Bug fixes
### PR changes

Others
### Description

pcard-72425
the fluid version of learning rate is or will be removed, so the unit test of set_lr_scheduler no longer needs to test it.
4c5ee7a27c5109572763193562ac53fa

### CODE REVIEW

**Review by XieYunshen — APPROVED** [review]
LGTM

**Review by jeff41404 — APPROVED** [review]
LGTM
